src/PanelTraits/Create.php

```diff
@@ -120,4 +144,94 @@ public function syncPivot($model, $data, $form = 'create')
             }
         }
     }
+
+    /**
+     * Create any existing one to one relations for the current model from the form data.
+     *
+     * @param Model $item The current CRUD model.
+     * @param array $data The form data.
+     * @param string $form Optional form type. Can be either 'create', 'update' or 'both'. Default is 'create'.
+     */
+    private function createOneToOneRelations($item, $data, $form = 'create')
+    {
+        $relationData = $this->getRelationDataFromFormData($data, $form);
+        $this->createRelationsForItem($item, $relationData);
+    }
+
+    /**
+     * Create any existing one to one relations for the current model from the relation data.
+     *
+     * @param Model $item The current CRUD model.
+     * @param array $formattedData The form data.
+     */
+    private function createRelationsForItem($item, $formattedData)
+    {
+        foreach ($formattedData['relations'] as $relationMethod => $relationData) {
```

Comment:
```
if (! isset($formattedData['relations'])) { }
````

seems prudent; I'd be inclined to check its array-ness too.

Reply:
Since this is part of a private function and the formatted data is also created in this class, this can be a little over protective...

Reply:
Yes, but even private functions get called with erroneous data.